Makefile

```diff
@@ -56,4 +56,4 @@ images:
 test:
 	@echo "Running testsuite"
 	@echo "-----------------"
-	-. venv/bin/activate && python -m tornado.testing discover
+	-. venv/bin/activate && flake8 remoteappmanager tests && python -m tornado.testing discover
```

Comment:
why not add another target `flake8` and then `test: flake8` should do the right  thing